generator/osm_source.cpp

```diff
@@ -144,7 +144,7 @@ void ProcessOsmElementsFromO5M(SourceReader & stream, std::function<void(OsmElem
   while (processorOsmElementsFromO5M.TryRead(element))
   {
     processor(std::move(element));
-    element.Clear();
+    element = {};
```

Comment:
```suggestion
    element.Clear();
```

@SiarheiFedartsou what was clang-tidy complaint about Clear? Why did you decide to change it with = {}?

Reply:
It was used after move(one line above) - in vast majority of cases it is a bug

Reply:
Could you please update this PR?
- put Clear() and write comment
- use normal { style (from new line) instead of KR-style

We will merge it and build beta version today.

Reply:
Here it is not a bug, right? Let's fix and move on. Other fixes here are important for the production release.